common/src/main/java/com/wynntils/features/user/overlays/map/MiniMapOverlayFeature.java

```diff
@@ -107,7 +107,7 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
 
             // TODO replace with generalized maps whenever that is done
             if (WebManager.isMapLoaded()) {
-                MapProfile map = WebManager.getMaps().get(0);
+                MapProfile map = WebManager.getMaps().peek();
```

Comment:
Why no .poll() isnt or do we need to contents in the future elsewhere?

Reply:
This is render loop. We need this every render.